### DISCUSSION TITLE
Saving a 2D plane instead of a 3D volume?

### DISCUSSION THREAD

**Discussion by kkwakwa** [nvladimus/npy2bdv#12] [Q&A]
Hi Nikita,

I was just wondering if it is possible to save a series of of multichannel 2D planes instead of a 3D volume? I tried defining a virtual stack with z=1 and then writing the plane into it, but that produced an HDF5 file that bigstitcher crashes when trying to open. Any suggestions?

**Comment by nvladimus** [comment]
Hi, @kkwakwa 
Can you share the file? It may be due to BigStitcher issue - it is known to ignore first and last plane of a stack, an old bug. Did you try to open it with BigDataViewer, for example?

**Reply by kkwakwa** [reply]
Hi @nvladimus 
I have attached the xml file as a text file. The actual HDF5 file is about 2.5GB, so I'd need to figure out how to send that file if you still think it'll be useful.

I tried just opening it in BigDataViewer and that seemed to work just fine, so I guess I can confirm that this is more than likely a BigStitcher issue. Are there any suggensions in that case? Can I make it explicit that the HDF5 file is a series of 2D tiles? If I save those tiles as tiffs and import them in to bigstitcher, manually place them and save out an HDF5 file, that works with no errors.

[Outdir.txt](https://github.com/nvladimus/npy2bdv/files/8125882/Outdir.txt)
l

**Reply by nvladimus** [reply]
Your xml file seems to contain 4 planes. You can share the dataset via Google drive or Dropbox, 2.5 GB is very manageable size.
BigStitcher does open 1-plane example file from its Wiki (http://preibischlab.mdc-berlin.de/BigStitcher/Grid_2d_h5_aligned.zip), you can look into its XML and check which parts of your dataset cause the crash. Its a bit of trial and error process, some bugs may lurk either in XML or in BigStitcher. You can also try opening in free Imaris Viewer, just to check if the issue is specific to BigStitcher.
